scaleway/helpers_vpc.go

```diff
@@ -40,3 +44,60 @@ func vpcAPI(m interface{}) (*vpc.API, error) {
 
 	return vpc.NewAPI(meta.scwClient), nil
 }
+
+const minVPCTimeOut = 10 * time.Second
+const PollVPCTimeOut = 30 * time.Second
+
+// retryVPCContext is a basic wrapper around StateChangeConf that will just retry with a poll interval
```

Comment:
Why is this change needed? Why can't we use a wait function around resources as we do in any other resource?

Reply:
We can ask a resource status in this case.